scripts/mainmenu_mapselector.js

```diff
@@ -118,14 +119,9 @@ class MapSelection {
 	 * @returns {Boolean} If all other modes are unchecked
 	 */
 	static areAllOtherModesUnchecked(selectedMode) {
-		const modes = this.gameModeData;
-		for (let i = 0; i < Object.keys(modes).length; i++) {
-			const curMode = modes[i];
-			if (curMode.filterButton !== null) {
-				if (curMode.filterButton.checked && curMode !== modes[selectedMode]) return false;
-			}
-		}
-		return true;
+		return !Object.entries(this.gameModeData)
```

Comment:
That exclamation mark is very "buried" and adds a bit of mental load when reading this code, is there not `.none` that could be used instead of the `.some`?

Reply:
This is a really good point, I agree. Unfortunately there's no `.none`. I guess we can use quantifier duality and negate the condition in a `.every`? It's actually quite frustrated that JS doesn't have such a method (iirc LINQ does) because yeah, it completely muddies up the readability.

Reply:
Yeah huh nevermind then. We could technically have it be `=== false` at the end to be more explicit though.